templates/index.html

```diff
@@ -2,31 +2,60 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-<div class="col-md-12">
-  <table class="table table-striped">
-    <thead>
-      <tr>
-        <th scope="col">id</th>
-        <th scope="col">Title</th>
-        <th scope="col">Status</th>
-        <th scope="col">Actions</th>
-      </tr>
-    </thead>
-    <tbody>
-      {% for item in items %}
-      <tr>
-        <th scope="row">{{ item['id']}}</th>
-        <td>{{ item['title']}}</td>
-        <td>{{ item['status'] }}</td>
-        <td>
-          <form class="form-inline" action="" method="PUT">
-            <button type="submit" class="btn btn-success" name="CompleteTask"><i class="fa fa-check" aria-hidden="true"></i>&nbspComplete</button>
-            <button type="submit" class="btn btn-danger" name="CompleteTask"><i class="fa fa-trash" aria-hidden="true" disabled></i>&nbspDelete</button>
-          </form>
-        </td>
-      </tr>
+<div class="Form">
+<div class="base">
+  <div class="board">
+    <div class="board-header">
+      <div class="left">
+        <div class="board-header-text">{{ boards[0]['name'] }}</div>
+      </div>
+    </div>
+    <div class="board-lists">
+      {% for my_lists in lists %}
+      <div class="board-list">
+        <div class="list-title">
+          {{ my_lists['name'] }}
+        </div>
+        {% if my_lists['id'] == "5f6885825920b36763183967" %}
```

Comment:
Because of these hardcoded list ids, the app will only work for your trello board, even if another user provides their own environment variables.

Given that you already get the cards from each list, you can hard code the list names, and then just place the appropriate cards in the right list - this will save us explicitly needing to fetch the lists at all

Reply:
removed the hardcoded list ids and set to a variable that is based on the environment variables as I could not get this to pull directly from the env